packages/core/src/Stores/client-store.js

```diff
@@ -1834,6 +1834,26 @@ export default class ClientStore extends BaseStore {
     }
 
     setAccounts(accounts) {
+        // TODO: delete this code when BE will be fixed
```

Comment:
Based on this [thread](https://deriv-group.slack.com/archives/C04ST0NNY5B/p1690807082474639) Kirill Sysoev asked to hide trading accounts that are not linked to any wallet, but I'm not sure it's a good idea and I need your advice )) Feel free to delete this code if you have the same doubts as me

Reply:
If we have to manipulate data like this, i think we should create a hook. What do you think @mahdiyeh-deriv

Reply:
@sergei-deriv We don't want to make any changes on the stores since we are using this observable many places and we don't want to add more logic to the stores anymore. So the best way in to relay on the API calls itself and make a hook out of them based on what you want.

Reply:
@thisyahlen-deriv @mahdiyeh-deriv I just deleted this code because this can lead to unexpected consequences. These orphan trading accounts don't interfere to application work and we can leave it for now and later BE will be fixed